Spanner/src/Operation.php

```diff
@@ -184,9 +184,13 @@ public function commitWithResponse(Session $session, array $mutations, array $op
      * @param string $transactionId The transaction to roll back.
      * @param array $options [optional] Configuration Options.
      * @return void
+     * @throws \InvalidArgumentException If the transaction is not yet initialized.
      */
     public function rollback(Session $session, $transactionId, array $options = [])
     {
+        if (is_null($transactionId)) {
+            throw new \InvalidArgumentException('Rollback failed: Transaction not initiated.');
```

Comment:
nit: let's add `use InvalidArgumentException` as an import so we don't need to use the `\`

Reply:
Done!